src/Jackett.Common/Content/custom.js

```diff
@@ -1327,15 +1327,17 @@ function updateSearchResultTable(element, results) {
                         if (keyword === "" || keyword === "+" || keyword === "-")
                             return;
                         var newKeyword;
+                        var pipedKeyword = '(' + keyword.substring(1).split('|').map(k => $.fn.dataTable.util.escapeRegex(k)).join('|') + ')';
                         if (keyword.startsWith("+"))
-                            newKeyword = $.fn.dataTable.util.escapeRegex(keyword.substring(1));
+                            newKeyword = pipedKeyword;
                         else if (keyword.startsWith("-"))
-                            newKeyword = "^((?!" + $.fn.dataTable.util.escapeRegex(keyword.substring(1)) + ").)*$";
+                            newKeyword = "^((?!" + pipedKeyword + ").)*$";
```

Comment:
You're right because NOT(A OR B) = NOT A AND NOT B.
Should we have a different logic for `|` in conjunction with `-` or just remove the `|` logic from here altogether?

Reply:
I think just remove it.